src/pages/publication/public_publication/PublicPublicationContent.tsx

```diff
@@ -128,10 +123,10 @@ const PublicPublicationContent: FC<PublicPublicationContentProps> = ({ publicati
       <Heading>{mainTitle}</Heading>
       {contributors && <PublicPublicationAuthors contributors={contributors} />}
       <StyledContentWrapper>
-        <StyledSidebar>
-          {publication.fileSet &&
-            publication.fileSet.files.map((file) => <PublicPublicationFile file={file} key={file.identifier} />)}
-        </StyledSidebar>
+        {publication.fileSet &&
```

Comment:
mente egentlig dette 😅 
```suggestion
        {
```